src/dotnet/commands/dotnet-tool/install/LocalizableStrings.resx

```diff
@@ -159,10 +159,17 @@
   <data name="InstallationSucceeded" xml:space="preserve">
     <value>You can invoke the tool using the following command: {0}
 Tool '{1}' (version '{2}') was successfully installed.</value>
+  </data>
+    <data name="LocalToolInstallationSucceeded" xml:space="preserve">
+    <value>You can invoke the tool using the following command with prefix: {0}
+Tool '{1}' (version '{2}') was successfully installed. Entry is added to the manifest file {3}.</value>
   </data>
```

Comment:
You can invoke the tool from this directory using the following command: dotnet dotnetsay
Tool 'dotnetsay' (version '2.1.4') was successfully installed. Entry is added to the manifest file /Users/williamli/Documents/temp/temp/.config/dotnet-tools.json.